bakery/src/tasks/check-feed.js

```diff
@@ -0,0 +1,45 @@
+const dedent = require('dedent')
+
+const task = ({ awsAccessKeyId, awsSecretAccessKey, bucketName, feedFileUrl }) => {
+  return {
+    task: 'check feed',
+    config: {
+      platform: 'linux',
+      image_resource: {
+        type: 'docker-image',
+        source: {
+          repository: 'openstax/cops-bakery-scripts'
```

Comment:
Is this image used because it has aws cli installed on it? There might be a leaner image with that installation. On the other hand, this may take better advantage of Docker layer caching depending on the image `openstax/cops-bakery-scripts` is derived from. Thoughts?

Reply:
Correct. In the past I did look around to see if there's a lean (and maintained) image with the `aws` cli but didn't find one (I used `mesosphere/aws-cli` at the time, but it's stale). That doesn't mean it wasn't a search failure on my part, though :). I also figured that since other tasks use the common image we'd benefit from image caching.